README.md

```diff
@@ -7,7 +7,7 @@ git clone --recursive https://github.com/poanetwork/cryptokitties-xdai-demo.git
 
 #### Initialize submodule and install dependencies
```

Comment:
> #### Initialize submodule and install dependencies	

Should be 
```
#### Install dependencies	
```

Reply:
Thanks, fixed https://github.com/poanetwork/cryptokitties-xdai-demo/pull/6/commits/8c730996d2bf6edcf3e91acf6a05b5889edffac4